.github/workflows/pytest-gpu.yaml

```
@@ -54,8 +54,7 @@ jobs:
       - name: Setup MCLI
         run: |
           set -ex
-          python -m pip install mosaicml-cli==0.3.0
-          mcli init --mcloud
+          python -m pip install mosaicml-cli==0.3.1
```

Actually can we not pin this?  The pin to 0.3.0 was just to workaround the breaking change.
```suggestion
          python -m pip install --upgrade mosaicml-cli
```

---

I think pinning to 0.3.0 is fine for now, I'll throw up a new PR when 0.4.0 is available